receiver/expvarreceiver/README.md

```diff
@@ -0,0 +1,49 @@
+# Expvar Receiver
+
+An Expvar Receiver scrapes metrics from [expvar](https://pkg.go.dev/expvar), 
+which exposes data in JSON format from an HTTP endpoint. The metrics are 
+extracted from the `expvar` variable [memstats](https://pkg.go.dev/runtime#MemStats), 
+which exposes various information about the Go runtime.
+
+> :construction: This receiver is in development and incomplete. It should not be used yet.
+
+## Configuration 
+
+### Default
+
+By default, without any configuration, a request will be sent to `http://localhost:8080/debug/vars` 
+every 60 seconds. The default configuration is achieved by the following:
+
+```yaml
+receivers:
+  expvar:
+```
+
+### Customising
+
+The following can be configured:
+- `http_client` - Configure the HTTP client for scraping the expvar variables. The full set of
```

Comment:
This is typically squashed into the config. [Examples](https://github.com/open-telemetry/opentelemetry-collector-contrib/search?q=HTTPClientSettings+%26%26+mapstructure)

Reply:
I initially thought that having the HTTP client in it's own config block would be slightly easier to understand, but I also appreciate the need for consistency. I've updated it so it's squashed like in the examples you linked.